packages/react-components/react-datepicker/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -246,51 +245,49 @@ export const CalendarMonth: React.FunctionComponent<CalendarMonthProps> = props
           </button>
         </div>
       </div>
-      <div {...arrowNavigationAttributes}>
```

Comment:
same as above, just moving the arrow navigation to the child and removing wrapping div